src/components/UploadPhoto/UploadPhoto.css

```diff
@@ -0,0 +1,41 @@
+.uploadPhotoContainer {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: inherit;  
+    justify-content: space-around;
+    width: inherit;
+}
+
+.photoContainer {
+    height: 300px;
+    width: 300px;
+}
+
+.photo {
+    width: 100%;
+    position: relative;
+}
+
+.uploadPhotoContainer>input {
```

Comment:
change  .uploadPhotoContainer>input to #photo-input

Reply:
ok